db/seeder.js

```diff
@@ -0,0 +1,125 @@
+  'https://leileihrr.s3.amazonaws.com/decor004.jpg',
+  'https://leileihrr.s3.amazonaws.com/decor005.jpg',
+  'https://leileihrr.s3.amazonaws.com/decor006.jpg',
+  'https://leileihrr.s3.amazonaws.com/kitchen001.jpg',
+  'https://leileihrr.s3.amazonaws.com/kitchen002.jpg',
+  'https://leileihrr.s3.amazonaws.com/kitchen003.jpg',
+  'https://leileihrr.s3.amazonaws.com/livingroom001.jpg',
+  'https://leileihrr.s3.amazonaws.com/livingroom003.jpg',
+  'https://leileihrr.s3.amazonaws.com/livingroom004.jpg',
+  'https://leileihrr.s3.amazonaws.com/livingroom005.jpg',
+  'https://leileihrr.s3.amazonaws.com/livingroom006.jpg',
+  'https://leileihrr.s3.amazonaws.com/livingroom007.jpg',
+  'https://leileihrr.s3.amazonaws.com/livingroom008.jpg',
+  'https://leileihrr.s3.amazonaws.com/livingroom009.jpg',
+  'https://leileihrr.s3.amazonaws.com/livingroom010.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors001.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors002.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors003.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors4.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors5.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors6.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors7.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors8.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors9.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors10.jpg',
+];
+const photoDesc = [
+  '',
+  'Spacious',
+  'Bright',
+  'Cheerful',
+  'Newly added',
+  'Comfortable',
+  'Cozy',
+  'Ensuite',
+  'Lofty',
+  'Warm',
+  'Roomy for the area',
+  'Accessible',
+  'Charming',
+  'Convenient',
+  'Functional',
+  'Comfy',
+  'Quaint',
+  'Soundproof',
+  'Renovated',
+  'Stylish',
+  '',
+];
+const verifiedOptions = [true, false];
+const data = [];
+
+const getRandomPhoto = (numPhotos) => {
+  const photoArray = [];
+  let photoId = 1;
+
+  for (let i = 0; i < numPhotos; i += 1) {
+    const url = photoUrl[Math.floor(Math.random() * Math.floor(photoUrl.length))];
+    const description = photoDesc[Math.floor(Math.random() * Math.floor(photoDesc.length))];
+    const verified = Math.floor(Math.random() * Math.floor(verifiedOptions.length));
+    const verifiedText = verified === 1 ? 'Photo Verified by Project Overnight' : '';
+
+    const newPhoto = {
+      photoId,
+      url,
+      description,
+      verified: verifiedText,
+    };
+    photoArray.push(newPhoto);
+    photoId += 1;
+  }
+  return photoArray;
+};
+
+// Generate x number of records for seeding data
+const createSampleData = (startingId, numRecords) => {
+  let id = startingId;
+  // An arbitrary number was chosen for the max in generating random
+  // Each UI page can have at max 5 elements so this allows for a small subset of data
+  const photoArray = getRandomPhoto(Math.floor(Math.random() * Math.floor(12)));
+
+  for (let i = 0; i < numRecords; i += 1) {
+    const newRecord = {
+      _id: id,
```

Comment:
This is a minor suggestion that involves line 102, 109, 113, and 119. You could cut down a little bit of code by assigning ‘i + 1’ to _id within the for loop. Then, your _id will still be incremented from 1 and you no longer need line 113, nor the startingId parameter of createSampleData!

Reply:
Thanks! I originally did that but changed it since I wanted to let the user decide what the id for their starting index for new inserts (e.g db already exists and they want to add 100 more records). However, I like this suggestion about cleaning up some of this. I'll be using a different insert instead.